docs/DeveloperGuide.md

```diff
@@ -262,27 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* *`    | user             | sort contact by name                                                        | see whose contact I have saved                                    |
+| `* *`    | user             | archive contact                                                             | hide less frequently used contacts without deleting them          |
+| `* *`    | user             | be alerted when a contact already exist                                     | avoid accidentally creating a duplicate                           |
+| `* *`    | user             | undo a command                                                              | fix a mistake I made                                              |
+| `* *`    | new user         | import all contact details into the app                                     | start using without manual setup                                  |
+| `* *`    | sales rep        | keep track of clients I have contacted by seeing when I last contacted them | avoid wasting time calling them again about the same product      |
+| `* *`    | sales rep        | view my most popular/active clients                                         | promote the new product                                           |
+| `* *`    | sales rep        | remember the client's preferred products                                    | recommend related products                                        |
+| `* *`    | sales rep        | add notes to client's contact                                               | keep track of my conversation with them                           |
+| `* *`    | sales rep        | group my clients by industry                                                | tell if sales are doing well in that industry among other metrics |
+| `* *`    | sales rep        | add tags to clients                                                         | categorize them                                                   |
+| `* *`    | sales rep        | keep note of my client's email addresses                                    | potentially send promotions or survey forms                       |
+| `* *`    | sales rep        | export a list of contact emails                                             | add them to a mailing list                                        |
+| `* *`    | sales rep        | add a tag to multiple clients                                               | tag the clients more easily                                       |
+| `*`      | experienced user | use keyboard shortcuts                                                      | navigate the app faster                                           |
+| `*`      | sales rep        | contact my client quickly from the app                                      | avoid typing numbers repeatedly on my phone                       |
```

Comment:
Shouldn't this be desktop app? Since it is stated In the target user profile.

Reply:
It is a desktop app. The implementation of this could be integration with WhatsApp Web/Telegram Web or some sort of VoIP.

Of course, we won't be doing it in this project. I can remove it if we decide that we want the user stories to represent things that we are more likely to actually do in the project.